python/ray/scripts/scripts.py

```diff
@@ -341,66 +305,69 @@ def dashboard(cluster_config_file, cluster_name, port, remote_port):
     help="manually specify the socket path of the raylet process")
 @click.option(
     "--temp-dir",
+    hidden=True,
     default=None,
     help="manually specify the root temporary dir of the Ray process")
-@click.option(
```

Comment:
Is it okay to delete this?

Reply:
According to this: https://docs.google.com/spreadsheets/d/1F6VZ1R_fgDzpRZjXI5_6gaUwW2CQKi431dCDTR-Vm8s/edit#gid=0&range=F12 yes

@raulchen maybe you can confirm?

Reply:
@wuisawesome can you leave this option alone in this PR? We have another PR to remove this option. Removing it requires more code changes. https://github.com/ray-project/ray/pull/10594/